mslib/msui/mpl_qtwidget.py

```diff
@@ -1269,13 +1319,17 @@ def draw_legend(self, img):
                                                xticks=[], yticks=[],
                                                label="ax2", zorder=0)
                 self.legax.patch.set_facecolor("None")
+                # self.legax.set_yticklabels("ax2", fontsize=32, minor=False)
```

Comment:
if comments are not needed for explainations, remove them

on all places.
if it is something you want to log ;) use logging.debug()

Reply:
Oka

> if comments are not needed for explainations, remove them
> 
> on all places.
> if it is something you want to log ;) use logging.debug()

Okay. Noted.

Reply:
> if comments are not needed for explainations, remove them
> 
> on all places.
> if it is something you want to log ;) use logging.debug()

Okay Noted.